clean-abap/CleanABAP.md

```diff
@@ -1880,15 +1880,19 @@ explains the reasoning behind this.
 Consider resolving complex constructions to a multi-step construction with the
 [Builder design pattern](https://en.wikipedia.org/wiki/Builder_pattern).
 
-#### Use descriptive names for multiple constructor methods
+#### Use descriptive names for multiple creation methods
 
-> [Clean ABAP](#clean-abap) > [Content](#content) > [Classes](#classes) > [Constructors](#constructors) > [This section](#use-descriptive-names-for-multiple-constructor-methods)
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Classes](#classes) > [Constructors](#constructors) > [This section](#use-descriptive-names-for-multiple-creation-methods)
+
+Good words to start creation methods are `new_`, `create_`, and `construct_`.
+People intuitively connect them to the construction of objects.
+They also add up nicely to verb phrases like `new_from_template`, `create_as_copy`, or `create_by_name`.
 
 ```ABAP
-CLASS-METHODS describe_by_data IMPORTING p_data TYPE any [...]
-CLASS-METHODS describe_by_name IMPORTING p_name TYPE any [...]
-CLASS-METHODS describe_by_object_ref IMPORTING p_object_ref TYPE REF TO object [...]
-CLASS-METHODS describe_by_data_ref IMPORTING p_data_ref TYPE REF TO data [...]
+CLASS-METHODS create_describe_by_data IMPORTING p_data TYPE any [...]
+CLASS-METHODS create_describe_by_name IMPORTING p_name TYPE any [...]
+CLASS-METHODS create_describe_by_object_ref IMPORTING p_object_ref TYPE REF TO object [...]
+CLASS-METHODS create_describe_by_data_ref IMPORTING p_data_ref TYPE REF TO data [...]
```

Comment:
"create describe" is a meaningless concatentation of verbs. Better "create descriptor"?

```
create_descriptor_by_data
create_descriptor_by_name
create_descriptor_by_obj_ref
create_descriptor_by_data_ref
```

Reply:
how about using `new`?

Reply:
also okay